rllib/BUILD

```diff
@@ -1062,7 +1063,7 @@ py_test(
 py_test(
     name = "policy/tests/test_trajectory_view_api",
     tags = ["policy"],
-    size = "small",
+    size = "large",
```

Comment:
This is kind of concerning in size, can we break it up so that it's at most medium?

Reply:
True, after getting rid of all enormous tests, we should probably tackle the large ones next.

Reply:
done, made the test shorter.